sotodlib/site_pipeline/make_atomic_filterbin_map.py

```diff
@@ -0,0 +1,521 @@
+                        help="Map without demodulation (e.g. with\
+                        a static HWP)")
+    parser.add_argument("--only_hits", action="store_true",
+                        help='Only create a hits map')
+
+    # detector position splits (fixed in time)
+    parser.add_argument("--det_in_out", action="store_true")
+    parser.add_argument("--det_left_right", action="store_true")
+    parser.add_argument("--det_upper_lower", action="store_true")
+
+    # time samples splits
+    parser.add_argument("--scan_left_right", action="store_true")
+
+    parser.add_argument("--ntod", type=int, )
+    parser.add_argument("--tods", type=str, )
+    parser.add_argument("--nset", type=int, )
+    parser.add_argument("--wafer", type=str,
+                        help="Detector set to map with")
+    parser.add_argument("--freq", type=str,
+                        help="Frequency band to map with")
+    parser.add_argument("--dec_ref", type=float,
+                        help="Decl. at which we will calculate the\
+                        reference R.A.")
+    parser.add_argument("--center_at", type=str)
+    parser.add_argument("--max_dets", type=int, )
+    parser.add_argument("--fixed_ftime", type=int, )
+    parser.add_argument("--min_dur", type=int, )
+    parser.add_argument("--site", type=str, )
+    parser.add_argument("--verbose", action="count", )
+    parser.add_argument("--quiet", action="count", )
+    parser.add_argument("--window", type=float, )
+    parser.add_argument("--dtype_tod", type=str)
+    parser.add_argument("--dtype_map", type=str)
+    parser.add_argument("--atomic_db", type=str,
+                        help='name of the atomic map database, will be\
+                        saved where this script is being run')
+    parser.add_argument("--hk_data_path",
+                        help='Path to housekeeping data')
+    return parser
+
+
+def _get_config(config_file):
+    return yaml.safe_load(open(config_file, 'r'))
+
+
+def get_ra_ref(obs, site='so_sat3', dec_ref=-40):
+    t_start = obs.obs_info.start_time
+    t_stop = obs.obs_info.stop_time
+    az = np.arange(
+        (obs.obs_info.az_center-1.0*obs.obs_info.az_throw),
+        (obs.obs_info.az_center+1.0*obs.obs_info.az_throw),
+        1.0)*putils.degree
+    el = obs.obs_info.el_center*putils.degree
+    csl = so3g.proj.CelestialSightLine.az_el(
+        t_start*np.ones(len(az)),
+        az, el*np.ones(len(az)),
+        site=site, weather='toco')
+    ra_, dec_ = csl.coords().transpose()[:2]
+    ra_ref_start = np.interp(dec_ref*putils.degree, dec_, ra_)
+    csl = so3g.proj.CelestialSightLine.az_el(
+        t_stop*np.ones(len(az)), az, el*np.ones(len(az)),
+        site=site, weather='toco')
+    ra_, dec_ = csl.coords().transpose()[:2]
+    ra_ref_stop = np.interp(dec_ref*putils.degree, dec_, ra_)
+    return ra_ref_start, ra_ref_stop
+
+
+def find_footprint(context, tod, ref_wcs, return_pixboxes=False, pad=1):
```

Comment:
It seems like the only thing this function actually does is to get_footprint and then buffer by 1 pixel. Am I missing something?  (I can tell this is descendant from a version that ran on multiple TODs... but this version does not.)

Given the recent fix in #1032, the buffer-by-1-pixel is probably not needed. So: can this function be removed, and coords.get_footprint(tod, ref_wcs) used alone?